packages/styles/scss/components/image-with-caption/image-with-caption.scss

```diff
@@ -30,6 +30,34 @@
   }
 }
 
+@mixin image-with-caption-inverse {
```

Comment:
Similar to the other PR, I don't think we should duplicate the other mixin entirely, we should treat as a modifier and only target the theme tokens.